packages/siwe/package.json

```diff
@@ -36,7 +36,7 @@
     "prettier": "^2.6.2"
   },
   "peerDependencies": {
-    "ethers": "^5.5.1"
+    "ethers": "^5.6.8 || ^6.0.8"
```

Comment:
any reason for minimum 5.6.8?

Reply:
Oh, awesome. I'll just move the conversation with the team since this might break stuff for some folks.